oarepo_vocabularies/ui/theme/assets/semantic-ui/js/oarepo_vocabularies_ui/form/components/CurrentLocationInformation.jsx

```diff
@@ -0,0 +1,124 @@
+// component to show the user where they currently are
+
+import React from "react";
+import { useAxios } from "../hooks/useAxios";
+import { Message, Breadcrumb } from "semantic-ui-react";
+import _ from "lodash";
+import { i18next } from "@translations/oarepo_vocabularies_ui/i18next";
+import { breadcrumbSerialization } from "../../utils";
+import { ErrorComponent } from "./Error";
+import PropTypes from "prop-types";
+
+const NewTopLevelItemMessage = () => (
+  <Message
+    icon="attention"
+    header={i18next.t("newItemMessage")}
+    size="tiny"
+    // don't understand how to reasonably set width for such a component in semantic!!
+    style={{ width: "68%" }}
+  />
+);
+
+const NewChildItemMessage = ({ newChildItemParentId }) => {
+  const { response, loading, error } = useAxios({
+    url: `/api/vocabularies/institutions/${newChildItemParentId}`,
+  });
+  return (
+    <React.Fragment>
+      {!loading && !error.message && (
+        <Message
+          icon="attention"
+          header={i18next.t("newChildItemMessage", {
+            item: response?.title?.cs,
+          })}
+          size="tiny"
+          content={
+            <Breadcrumb
+              icon="right angle"
+              sections={[
+                ..._.reverse(
+                  breadcrumbSerialization(
+                    response?.hierarchy?.ancestors_or_self
+                  )
+                ),
+                { key: "new", content: i18next.t("newItem"), active: true },
+              ]}
+            />
+          }
+          // don't understand how to reasonably set width for such a component in semantic!!
+          style={{ width: "68%" }}
+        />
+      )}
+      {error.message && <ErrorComponent error={error} />}
+    </React.Fragment>
+  );
+};
+
+const EditMessage = ({ vocabularyRecord }) => {
+  const {
+    hierarchy: { level, ancestors_or_self },
+  } = vocabularyRecord;
+  if (level === 1)
+    return (
+      <Message
+        icon="attention"
+        header={i18next.t("editTopLevelItemMessage")}
+        size="tiny"
+        // don't understand how to reasonably set width for such a component in semantic!!
+        style={{ width: "68%" }}
+      />
+    );
+  return (
+    <Message
+      icon="attention"
+      header={i18next.t("editChildItemMessage")}
+      size="tiny"
+      content={
+        <Breadcrumb
+          icon="right angle"
+          sections={_.reverse(breadcrumbSerialization(ancestors_or_self))}
+        />
+      }
+      // don't understand how to reasonably set width for such a component in semantic!!
```

Comment:
In Semantic, width is determined by a parent element, and/or element's less definitions & variables.
See message's customizable [definition](https://github.com/Semantic-Org/Semantic-UI-LESS/blob/master/definitions/collections/message.less) and [variables](https://github.com/Semantic-Org/Semantic-UI-LESS/blob/master/themes/default/collections/message.variables) for reference.

If there is no suitable `@width` variable to tune, you can add/override styles from definition and set custom variables in `theme/assets/semantic-ui/less/oarepo_vocabularies_ui/theme/collections/message.{overrides,variables}`

Avoid setting percentage-based fixed `width` on such elements (not very responsive), it is better to clamp it using `min/max-width` or [`clamp()`](https://developer.mozilla.org/en-US/docs/Web/CSS/clamp)